internal/http/proxy/proxy.go

```diff
@@ -23,30 +24,28 @@ type TracingConfig struct {
 	RedactedHeaders []string `yaml:"redacted_headers"`
 }
 
-func NewProxy(cfg Config, blockFieldSuggestions *block_field_suggestions.BlockFieldSuggestionsHandler) (*httputil.ReverseProxy, error) {
+func NewProxy(cfg Config, blockFieldSuggestions *block_field_suggestions.BlockFieldSuggestionsHandler, excludeSubgraphErrors *exclude_subgraph_errors.ExcludeSubgraphErrors) (*httputil.ReverseProxy, error) {
 	target, err := url.Parse(cfg.Host)
 	if err != nil {
 		return nil, err
 	}
 	proxy := &httputil.ReverseProxy{
 		Rewrite: func(r *httputil.ProxyRequest) {
 			r.Out.Header["X-Forwarded-For"] = r.In.Header["X-Forwarded-For"]
+			r.Out.Header.Set("Accept-Encoding", "-") //As compression has no direct benefit for us within our cloud setup, to get this working the modifyResponse implementation need to be adapted to handle compression
```

Comment:
This is technically an invalid value https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding . Could you test if this works if you explicitly drop the `Accept-Encoding` header?

Reply:
yeaa good idea thats a better solution indeed